bots/images/scripts/lib/atomic.install

```diff
@@ -189,11 +191,11 @@ class AtomicCockpitInstaller:
 
     def package_basenames(self, package_names):
         """ convert a list of package names to a list of their basenames """
-        return filter(lambda s: not s is None, map(lambda s: self.package_basename(s), package_names))
+        return [s for s in [self.package_basename(s) for s in package_names] if s is not None]
```

Comment:
Why is this change necessary. It's harder to read, no?

Reply:
Interesting, I actually find the list comprehensions easier to read (and 2to3 suggests changing that); but I suppose that's a matter of what you are used to. In python3, `filter()` returns an iterator, not a list, so alternatively we can keep the filter/map and wrap the whole thing into a `list()`.

I did the latter now, and as a bonus dropped the unnecessary inner lambda.